.github/workflows/ci.yaml

```diff
@@ -36,6 +37,25 @@ jobs:
             gromacs-version: "2018.6"
           - os: macOS-latest
             gromacs-version: "2020.6"
+          - os: macOS-latest
+            python-version: 3.8
+          - os: macOS-latest
+            python-version: 3.7
+          - os: macOS-latest
+            python-version: 3.6
+        include:
+          - os: macOS-latest
+            python-version: 3.9
+            gromacs-version: "2021.1"
+          - os: ubuntu-latest
+            python-version: 3.7
+            gromacs-version: "2021.1"
+          - os: ubuntu-latest
+            python-version: 3.8
+            gromacs-version: "2021.1"
+          - os: ubuntu-latest
+            python-version: 3.9
+            gromacs-version: "2021.1"
```

Comment:
These includes are superfluous because ubuntu 2021.1 with 3.7, 3.8, 3.9 is already included in the matrix.
```suggestion
```